_l10n/md2html.py

```diff
@@ -0,0 +1,197 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
Given this is reused with NVDA, it's going to be awful to keep it in sync. Do you think it's worth turning it into a separate submodule, and it including it both here and in NVDA?

Reply:
can you make one based directly off the l10n submodule in NVDA? we can add any necessary changes for the the add-on template to it separately

Reply:
OK, I'll try to do it in the next days

Reply:
The repo is at

https://github.com/nvdaes/nvdaL10n